src/beamerinnerthememoloch.dtx

```diff
@@ -204,7 +204,7 @@
   \fi
 }
 \def\titlepage{%
-  \usebeamertemplate{title page}
+  \usebeamertemplate{title page}\ifx\insertthanks\@empty\else\@thanks\fi
```

Comment:
`\insertthanks` isn't defined, no need to test. Maybe just keep the beamer definition of the title page? `\def\titlepage{\usebeamertemplate*{title page}\@thanks}`